backend/routers/plugins.py

```
@@ -109,3 +116,129 @@ def get_plugin_money_made(plugin_id: str):
         'type2': type2,
         'type3': type3
     }
+
+
+@router.post('/v2/plugins/enable')
+def enable_plugin_endpoint(plugin_id: str, uid: str = Depends(auth.get_current_user_uid)):
```

Ruff

Redefinition of unused `enable_plugin_endpoint` from line 25

(F811)